lock/src/main/java/org/infinispan/lock/impl/ClusteredLockModuleLifecycle.java

```diff
@@ -66,11 +66,14 @@ public void cacheManagerStarted(GlobalComponentRegistry gcr) {
       ClusteredLockManagerConfiguration config = extractConfiguration(gcr);
       GlobalConfiguration globalConfig = gcr.getGlobalConfiguration();
 
-      internalCacheRegistry.registerInternalCache(CLUSTERED_LOCK_CACHE_NAME, createClusteredLockCacheConfiguration(config, globalConfig),
-            EnumSet.of(InternalCacheRegistry.Flag.EXCLUSIVE));
-
-      CompletableFuture<CacheHolder> future = startCaches(cacheManager);
-      registerClusteredLockManager(gcr, future, config);
+      if (globalConfig.isClustered()) {
+         internalCacheRegistry.registerInternalCache(CLUSTERED_LOCK_CACHE_NAME, createClusteredLockCacheConfiguration(config, globalConfig),
+               EnumSet.of(InternalCacheRegistry.Flag.EXCLUSIVE));
+         CompletableFuture<CacheHolder> future = startCaches(cacheManager);
+         registerClusteredLockManager(gcr, future, config);
+      } else {
+         log.info("Clustered locks are necessary for clustered infinispan");
```

Comment:
INFO message is a bit confusing, maybe something like "Configuration is not clustered, clustered locks are disabled" or something like that.

Reply:
ok